src/elements/play-pen/play-pen.ts

```diff
@@ -409,4 +447,13 @@ export class PlayPen extends LitElement {
       this._uploaded =
         this.#bundleStore?.upload(this._bundle) ?? Promise.resolve({})
   }
+
+  #assetsUpdated = () => {
```

Comment:
if you don't mind, please type methods and props :pray: this really help me read the code and verify intent.

Reply:
Can do, though does it clarify to type a void lambda like this one?  It adds a lot of extra noise:

```ts
  #assetsUpdated: () => void = () => { }
```